File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    
    str_ab = str(a) + str(b)
    str_ba = str(b) + str(a)
    
    if int(str_ab) > int(str_ba):
        answer = int(str_ab)
    else:
        answer = int(str_ba)
              
    return answer

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        
        String str_ab = Integer.toString(a) + Integer.toString(b);
        String str_ba = Integer.toString(b) + Integer.toString(a);
        int num_ab = Integer.parseInt(str_ab);
        int num_ba = Integer.parseInt(str_ba);
        
        if(num_ab > num_ba){
            return num_ab;
        }
        else{
            return num_ba;
        }

    }
}